### DISCUSSION TITLE
Laravel Precognition v0.5.5 - Release notes

### DISCUSSION THREAD

**Discussion by sanfair** [laravel/framework#51795] [Q&A] [answered]
There was a new release on the 4th of June that exists in [npmjs.com](https://www.npmjs.com/package/laravel-precognition/v/0.5.5)
And @dependabot already created PR for version update
However, I can not find any Release notes or Changelog.

It seems that this version was released differently than usual and was only released in [npm](https://www.npmjs.com/package/laravel-precognition/v/0.5.5?activeTab=versions) but not in the [GitHub](https://github.com/laravel/precognition/releases) repo

@driesvints could you please take a look

**Comment by Sophist-UK** [comment]
I appreciate that you are concerned that this may be a malicious update, however [here](https://github.com/laravel/precognition/commit/2b5483b121d016a8169909f6f03d7ed535f9e082) is the commit by Taylor Otwell himself producing 0.5.5. So as far as I can tell, this is an OK release.

**Comment by driesvints** [comment, marked as answer]
I added the release notes now 👍

**Reply by sanfair** [reply]
Thank you!
